.travis.yml

```diff
@@ -5,6 +5,7 @@ cache: bundler
 rvm:
 - jruby-1.7.25
 script:
-- bundle exec rspec spec
+- bundle exec rspec spec --order rand
```

Comment:
@suyograo - unfortunately, I was completely stumped by this bit of 'protection' from Travis CI. `Similarly, we do not provide these values to untrusted builds, triggered by pull requests from another repository` in this [link](https://docs.travis-ci.com/user/environment-variables/). I kept getting config errors.

This is because of our PR from private fork policy.

The only thing I can do is to add some better unit tests that verify the fix - however I will backburner that for later.